Source/WebKit/UIProcess/ios/forms/WKFileUploadPanel.mm

```diff
@@ -353,6 +370,7 @@ @implementation WKFileUploadPanel {
     RetainPtr<WKFileUploadMediaTranscoder> _mediaTranscoder;
 #endif
     RetainPtr<UIImagePickerController> _imagePicker;
+    RetainPtr<PHPickerViewController> _photoPicker;
```

Comment:
I really have no idea what I should name this

Reply:
Maybe we rename `_imagePicker` since it's really just about the camera flow now.